.changeset/cuddly-lands-work.md

```diff
@@ -0,0 +1,5 @@
+---
+'@spectrum-web-components/card': patch
+---
+
+fix(card) - added missing sp-popover dependency in sp-card dependency tree
```

Comment:
This is still reading like a commit message. Can we get in the habit of writing out changesets messages as detailed changelog notes?

Reply:
Thanks @nikkimk for sharing.